### PR TITLE
Fix query string generation for noGlobals option

### LINKED ISSUE #92: `noGlobals` won't work on a url that already has a query string
`url.format()` ignores `query` if `search` already exists.

### PR DESCRIPTION
**Pull request by scottgonzalez**
Fixes #92

### CODE REVIEW

**Comment by tkellen** [comment]
Would you be willing to add a test for this?

**Comment by scottgonzalez** [comment]
Sure, I could add a querystring to the qunit3.html test file and then have an assertion that just uses `QUnit.urlParams` to verify it still exists. Does that sound good?

**Comment by tkellen** [comment]
:+1:

**Comment by scottgonzalez** [comment]
I added a test for this, but the tests don't complete for me even in master, so I'm not sure if the expected value is correct.

**Comment by tkellen** [comment]
oy vey.

**Comment by scottgonzalez** [comment]
Fixed the existing failing tests in #95 and rebased this.

**Comment by jzaefferer** [comment]
With #95 merged, this now passes as well. Travis passes, will merge this.

**Comment by tkellen** [comment]
Thanks @jzaefferer !

File: Gruntfile.js [hunk 1]
```diff
@@ -82,6 +85,7 @@ module.exports = function(grunt) {
       'test/qunit1.html': 3,
       'test/qunit2.html': 3,
       'http://localhost:9000/test/qunit1.html': 2,
+      'http://localhost:9000/test/qunit3.html?foo=bar&noglobals=': 1,
```
Comment (Arkni) [inline Gruntfile.js]: This is not valid any more in `QUnit 1.22.0`. Caused by https://github.com/jquery/qunit/commit/a5806f1178b02b9655f19569ac21adf510d8c62f

So the url should be either:
* `http://localhost:9000/test/qunit3.html?foo=bar&noglobals`
* or `http://localhost:9000/test/qunit3.html?foo=bar&noglobals=truthy-value`

Comment (Arkni) [inline Gruntfile.js]: Opened #114 to fix that.

**Pull request merged**